src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
+        System.out.println(ball.getRandomBall());
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall());
     }
```

Comment:
[Don’t create several instances of the same class if you can use only one instance for your purposes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes)
let's use cycle here, so we can easily print 3, 10 or 100 balls
also, `ball` is not the best name for variable with type `Lottery`)